study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/study/viewFetalStudyActiveTask.jsp

```diff
@@ -12,24 +21,26 @@
   <input type="hidden" value="${actionPage}" id="actionPage" name="actionPage">
   <input type="hidden" value="${currentPage}" id="currentPageId" name="currentPage">
   <div class="pt-lg">
-    <div class="gray-xs-f mb-sm">Activity Short Title or Key
+    <div class="gray-xs-f mb-sm">Activity short title or key
       <small>(50 characters max)</small>
       <span
           class="requiredStar"> *
       </span>
       <span class="ml-xs sprites_v3 filled-tooltip"
             data-toggle="tooltip"
-            title="This must be a human-readable activity identifier and unique across all activities of the study.Note that this field cannot be edited once the study is Launched."></span>
+            title="A human-readable identifier that must be unique across all activities of the study. Allowed characters are lowercase alphabets (a-z), digits (0-9), _ (underscore) and -(minus)."></span>
```

Comment:
Alphabets is not the correct term - get help from tech writer

Reply:
Fixed review comment